modules/filestore/store.go

```diff
@@ -20,6 +22,8 @@ type Module struct {
 	store   FileStore
 	enabled bool
 	auth    *auth.Module
+	crud    *crud.Module
+	event   *eventing.Module
```

Comment:
Where are you setting this variable? Should it be received in the `Init` function?
Also we'll use the interface instead of the actual eventing module

Reply:
even though i am not able to import use event field in filestore module, as it give rise to import cycle not allowed error

![Screenshot from 2019-12-17 22-43-02](https://user-images.githubusercontent.com/24411676/71018553-b35a5a80-211e-11ea-8446-15ed88fb901d.png)

Reply:
so how should i call eventing functions in filestore module

Reply:
Make an interface!!!!